packages/editor/src/components/inserter/menu.js

```diff
@@ -33,7 +33,6 @@ import { withInstanceId, compose, withSafeTimeout } from '@wordpress/compose';
 import BlockPreview from '../block-preview';
 import BlockTypesList from '../block-types-list';
 import ChildBlocks from './child-blocks';
-import InserterResultsPortal from './results-portal';
```

Comment:
It looks like `InserterResultsPortal` is no longer used. Can we remove all references and the obsolete file?

Reply:
Removed with https://github.com/WordPress/gutenberg/pull/10209/commits/3bea61e1881199603f7f68b417389715c6213f07